src/core/connectors.js

```diff
@@ -1553,6 +1553,13 @@ const connectors = [{
 	],
 	js: 'connectors/epidemicsound.js',
 	id: 'epidemicsound',
+}, {
+	label: 'REKT.NETWORK',
```

Comment:
Could you change the label to "Rekt Network"? They have this name in Twitter and in their website (but it's changed to "Rekt FM" after the website is loaded, but I think it's a bug).